tests/LivewireDatatableClassTest.php

```diff
@@ -71,13 +70,102 @@ public function it_can_order_results()
         $this->assertEquals('Advanced beet growing', $subject->results->getCollection()[1]->subject);
 
         $subject->forgetComputed();
-        $subject->sort = 1;
-        $subject->direction = true;
+        $subject->sort = ['1|asc'];
 
         $this->assertEquals('Advanced beet growing', $subject->results->getCollection()[0]->subject);
         $this->assertEquals('Beet growing for noobs', $subject->results->getCollection()[1]->subject);
     }
 
+    /** @test */
+    public function it_can_order_results_for_multiple_columns_using_column_index()
+    {
+        factory(DummyModel::class)->create(['subject' => 'Beet growing for noobs', 'category' => 'A']);
+        factory(DummyModel::class)->create(['subject' => 'Advanced beet growing', 'category' => 'A']);
+        factory(DummyModel::class)->create(['subject' => 'Advanced beet growing', 'category' => 'B']);
+        factory(DummyModel::class)->create(['subject' => 'Beet growing for noobs', 'category' => 'B']);
```

Comment:
uou can use createMany here factory(DummyModel::class)->createMany([ ['subject' => 'Beet growing for noobs', 'category' => 'A'], ['subject' => 'Advanced beet growing', 'category' => 'A'],...]);